pandas/tests/indexes/test_indexing.py

```diff
@@ -332,3 +332,12 @@ def test_get_indexer_non_unique_multiple_nans(idx, target, expected):
     axis = Index(idx)
     actual = axis.get_indexer_for(target)
     tm.assert_numpy_array_equal(actual, expected)
+
+
+def test_get_indexer_non_unique_nans_in_object_dtype_target():
+    idx = Index([1.0, 2.0])
+    target = Index([1, np.nan], dtype="object")
```

Comment:
can you parametrize using nulls_fixture instead of np.nan

Reply:
Done.